ecosystem/typescript/sdk_v2/tests/unit/multi_ed25519.test.ts

```diff
@@ -119,12 +119,12 @@ describe("MultiSignature", () => {
   it("should throws exception when creating a bitmap with wrong bits", async () => {
     expect(() => {
       MultiEd25519Signature.createBitmap({ bits: [32] });
-    }).toThrow("Cannot have a signature larger than 31.");
```

Comment:
that was changed to this in https://github.com/aptos-labs/aptos-core/pull/10290/files#diff-fa35ec805433effb2c47fca68711344abbcbc0f5f15786cae18b1db7143e454dR122

Reply:
argh I thought I changed this, guess I accidentally left it in, my bad